client/driver/executor/executor.go

```diff
@@ -188,11 +216,22 @@ func (e *UniversalExecutor) UpdateLogConfig(logConfig *structs.LogConfig) error
 	return nil
 }
 
+func (e *UniversalExecutor) UpdateTask(task *structs.Task) error {
+	e.ctx.Task = task
+	if e.lro != nil && task.LogConfig != nil {
+		e.lro.MaxFiles = task.LogConfig.MaxFiles
+		e.lro.FileSize = int64(task.LogConfig.MaxFileSizeMB * 1024 * 1024)
+	}
+	if e.lre != nil && task.LogConfig != nil {
```

Comment:
@dadgar Yeah you are right. I was thinking we don't have validation around it if the users submitted the jobs via the http api.